server/routes/contacts/add/contact-confirmation/contactConfirmationController.ts

```diff
@@ -55,6 +59,18 @@ export default class ContactConfirmationController implements PageHandler {
     return res.redirect(`/prisoner/${journey.prisonerNumber}/contacts/search/${journeyId}`)
   }
 
+  private async formattedGender(contact: ContactDetails, user: Express.User) {
```

Comment:
You don't need this at all. `genderDescription` is already populated with the reference data description. Plus this is still capitalising.

Reply:
Changes reverted